src/main/java/org/testcontainers/containers/PathUtils.java

```diff
@@ -0,0 +1,29 @@
+package org.testcontainers.containers;
+
+/**
+ * Utils class for path normalization.
+ *
+ * @author Vladimir Rogach
+ */
+public class PathUtils {
+
+    private PathUtils() {
+    }
+
+    /**
+     * Removes leading slash under windows from "/C:/work" so
+     * the path will match the format expected by docker.
+     *
+     * @param binding path
+     * @return normalized path for docker
+     */
+    public static String normalizeBindingPath(String path) {
+        if(path != null 
```

Comment:
The slash comes from the URL (Java resource path is actually an URL). Use a `File` constructor + `getAbsolutePath`. It will remove the extra slashes.

Also, we need to introduce this normalization in all places, e.g. in `executeScript` and for `TarantoolContainer` as well.

Reply:
>> The slash comes from the URL (Java resource path is actually an URL). Use a File constructor + getAbsolutePath. It will remove the extra slashes.

This does not work on windows - I've checked your suggestion and added unit test for this.

Reply:
> The slash comes from the URL (Java resource path is actually an URL). Use a File constructor + getAbsolutePath. It will remove the extra slashes.

This method does not work for unix paths under windows - it explicitly adds drive letter to path. For example - ```/dummy``` will be converted to ```C:/dummy```

Reply:
I've also refactored path normalization over all project.